### PR TITLE
Add tests for issue 56 and 60

### PR DESCRIPTION
**Pull request by nomisRev**
This PR adds two failing test cases discovered after discussion with @traceyyoshima in https://github.com/openrewrite/rewrite-kotlin/pull/55.

Relates to #56 and #60

### CODE REVIEW

File: src/test/java/org/openrewrite/kotlin/tree/LambdaTest.java [hunk 1]
```diff
@@ -67,4 +69,34 @@ fun inputValues ( ) : List < Pair < String , Any ? > > {
           )
         );
     }
+
+    @Issue("https://github.com/openrewrite/rewrite-kotlin/issues/60")
+    @Test
+    @Disabled
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/kotlin/tree/LambdaTest.java:75]: We could adopt `@ExpectedToFail` here from junit-pioneer; that way the test will always run, and fail only when the issue is fixed, to signify that we can update and close the associated issue. Optional, but potentially nice to have.

Comment (nomisRev) [reply src/test/java/org/openrewrite/kotlin/tree/LambdaTest.java:75]: Sounds like a good idea! Shall I do the same for test cases added in https://github.com/openrewrite/rewrite-kotlin/pull/55 and https://github.com/openrewrite/rewrite-kotlin/pull/57?

They're a bit unrelated to this PR.. I can also split it if preferred ☺️

Comment (timtebeek) [reply src/test/java/org/openrewrite/kotlin/tree/LambdaTest.java:75]: Fine to do that here & Thanks for the quick response!

**Comment by nomisRev** [comment]
@timtebeek I updated the PR, but it required me to introduce a new dependency. The repo uses `release.latest` but that wasn't working for `junit-pioneer` so I had to specify `2.0.0` explicitly.

File: src/test/java/org/openrewrite/kotlin/tree/LambdaTest.java [hunk 2]
```diff
@@ -15,7 +15,10 @@
  */
 package org.openrewrite.kotlin.tree;
 
+import org.junit.jupiter.api.Disabled;
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/kotlin/tree/LambdaTest.java:18]:
```suggestion
```

File: src/test/java/org/openrewrite/kotlin/tree/MethodDeclarationTest.java [hunk 3]
```diff
@@ -17,6 +17,7 @@
 
 import org.junit.jupiter.api.Disabled;
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/kotlin/tree/MethodDeclarationTest.java:18]:
```suggestion
```

File: src/test/java/org/openrewrite/kotlin/tree/VariableDeclarationTest.java [hunk 4]
```diff
@@ -17,6 +17,7 @@
 
 import org.junit.jupiter.api.Disabled;
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/kotlin/tree/VariableDeclarationTest.java:18]:
```suggestion
```

**Review by timtebeek — APPROVED** [review]
I _think_ we can clean up these imports, but haven't checked; let's try & merge if this works.

**Comment by nomisRev** [comment]
Oops, missed cleaning up the imports indeed. Normally I use `cmd+opt+l` in IntelliJ but that seems to format to a completely different style than I have.

Do you have any `.editorconfig` configured in OpenRewrite projects? That could make it easier for contributors to stick to the same style without changing IntelliJ configurations.

**Comment by timtebeek** [comment]
> Oops, missed cleaning up the imports indeed. Normally I use `cmd+opt+l` in IntelliJ but that seems to format to a completely different style than I have.

No worries; we're working on automating these changes to lighten the review load. :)

> Do you have any `.editorconfig` configured in OpenRewrite projects? That could make it easier for contributors to stick to the same style without changing IntelliJ configurations.

We often do; don't know if that's in rewrite-kotlin already, but we have a few scattered around and plans to merge this towards openrewrite/rewrite eventually.

**Pull request merged**